src/components/Input/index.js

```diff
@@ -164,9 +175,32 @@ class Input extends Component {
           }}
           rows={1}
         />
+        <div
+          style={{
+            display: 'flex',
+          }}
+        >
+          <button
```

Comment:
It would be better if the button with the SVG are in a new component (`SendButton` for example). It will isolated the SVG in a new component which is cleaner.

Reply:
Alright make sense!